app/controllers/application_controller.rb

```diff
@@ -9,12 +9,20 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   rescue_from Blacklight::Exceptions::RecordNotFound do
```

Comment:
could this be `rescue_from ActiveSupport::etc, Blacklight::etc`? Might need to be split onto 2 lines because the exceptions are long but still might be a bit tighter.

Reply:
@hackartisan I just pushed up a changes along this line, but hopefully easier on the eyes and simpler to expand in the future.